src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3154,29 +3202,29 @@ internal abstract partial class MemberReferenceExpression : Operation, IMemberRe
         /// <summary>
         /// Referenced member.
         /// </summary>
-        public ISymbol Member { get; }
+        public virtual ISymbol Member { get; }
```

Comment:
Is `Member` overridden?

Reply:
Not anymore, removed.

---
In reply to: [143323698](https://github.com/dotnet/roslyn/pull/22056#discussion_r143323698) [](ancestors = 143323698)